.yamato/_run-all.yml

```diff
@@ -35,7 +37,7 @@ all_package_tests:
 {% for editor in test_editors -%}
 {% for platform in test_platforms -%}
 {% for project in projects -%}
-    - .yamato/package-tests.yml#test_{{ projects.first.packages.first.name }}_{{ editor }}_{{ platform.name }}
+    - .yamato/package-tests.yml#test_{{ projects.first.packages.second.name }}_{{ editor }}_{{ platform.name }}
```

Comment:
Sort of flying blind without strong Yamato knowledge, but I'm not seeing where the project induction variable is getting used inside its loop

Reply:
Good call will fix